configs/xcache-consistency-check/xcache-consistency-check/default.cfg

```diff
@@ -0,0 +1,17 @@
+[Main]
+
+#(REQUIRED) Where are the .root files to analyze
+#path = /xcache-root/
+
+#(REQUIRED) Where to store the DataBase
+db = /var/lib/xcache-consistency-check/db.sql
+
+# Where to store the logs
+logfile = /var/log/xcache-consistency-check/log.out
```

Comment:
Seems like overkill to have another dir just for this. Perhaps throw this in `/var/log/xrootd/xcache-consistency-check.log`? Or write to `/var/log/xcache-consistency-check.log`

Reply:
If there are any other log files, it should probably get it's on log directory.  Otherwise (if it's the only log file related to the package), then either way is probably fine.

Though tbh i think i'd suggest something more like `/var/log/xcache/consistency-check.log`

Reply:
I prefer `/var/log/xrootd/` over `/var/log/xcache/` because all of the XCache logs get dropped into the former.

Reply:
Ah, i was just going off the package name.  I would have expected all the xcache logs to go under their own subdir, but if they are already going under /var/log/xrood, then yeah it makes sense to be consistent at least.